postgres-appliance/configure_spilo.py

```diff
@@ -256,26 +261,29 @@ def get_placeholders(provider):
     placeholders.setdefault('WALE_BACKUP_THRESHOLD_MEGABYTES', 1024)
     placeholders.setdefault('WALE_BACKUP_THRESHOLD_PERCENTAGE', 30)
     placeholders.setdefault('WALE_ENV_DIR', os.path.join(placeholders['PGHOME'], 'etc', 'wal-e.d', 'env'))
+    placeholders.setdefault('USE_WALE', False)
 
     if provider in (PROVIDER_AWS, PROVIDER_GOOGLE):
-        placeholders.setdefault('USE_WALE', True)
-        if provider == PROVIDER_AWS:
-            placeholders.setdefault('WAL_S3_BUCKET', 'spilo-example-com')
-        elif provider == PROVIDER_GOOGLE:
-            placeholders.setdefault('WAL_GCS_BUCKET', 'spilo-example-com')
+        if provider == PROVIDER_AWS and 'WAL_S3_BUCKET' in placeholders:
+            placeholders['USE_WALE'] = True
+        elif provider == PROVIDER_GOOGLE and 'WAL_GCS_BUCKET' in placeholders:
+            placeholders['USE_WALE'] = True
             placeholders.setdefault('GOOGLE_APPLICATION_CREDENTIALS', '')
+
         # Kubernetes requires a callback to change the labels in order to point to the new master
         if USE_K8S:
             placeholders.setdefault('CALLBACK_SCRIPT', '/callback_role.py')
         elif provider == PROVIDER_AWS:  # AWS specific callback to tag the instances with roles
             placeholders.setdefault('CALLBACK_SCRIPT', 'patroni_aws')
 
     else:  # avoid setting WAL-E archive command and callback script for unknown providers (i.e local docker)
-        placeholders.setdefault('USE_WALE', False)
         placeholders.setdefault('CALLBACK_SCRIPT', '')
```

Comment:
For K8S, the callback should be set to /callback_role no matter which provider we are using (it can be neither Google nor AWS in case of a minikube).